src/pages/Integrations/SplunkSetup/SplunkSetupPage.tsx

```diff
@@ -0,0 +1,366 @@
+            return result;
+        } catch (error) {
+            const message = 'Failed to update event type with behavior group';
+            error.message = message;
+            throw error;
+        }
+    };
+
+    async function startAutomation() {
+        try {
+            const group = await createGroup();
+            await addUserToGroup(group.uuid);
+            await addRoleToGroup(group.uuid);
+            const integration = await createIntegration();
+            const behavior = await createBehaviorGroup();
+            await associateIntegrationWithBehaviorGroup(behavior.id, integration.id);
+            for (const eventType of Constants.EVENT_TYPES_IDS_ARR) {
+                const behaviorGroupsData = await fetchBehaviorGroupsFromEventType(eventType);
+                const behaviorGroupsArr = parseBehaviorGroups(behaviorGroupsData);
+                behaviorGroupsArr.push(behavior.id);
+                await updateEventType(eventType, behaviorGroupsArr);
+            }
+
+            const message = 'Setup is done!';
+            setAutomationLogs(prevLogs => `${prevLogs}${message}`);
+        } catch (error) {
+            if (error.response) {
+                console.log(error.message);
+            } else if (error.request) {
+                console.log(error);
+            }
+            // Send issue Alert to user
+            throw error;
+        }
+    }
+
+    return (
+        <>
+            <PageHeader>
+                <PageHeaderTitle title={Messages.pages.splunk.page.title} />
+            </PageHeader>
+            <Flex>
+                <Main className='c-splunk-setup-display'>
+                    <Section>
+                        <FlexItem>
+                            <Form>
+                                <FormGroup
+                                    label="Server hostname/IP Address and port (hostname:port)"
+                                    labelIcon={<Popover
+                                        headerContent={<div>
+                                            The server <b>hostname/IP Address</b> and <b>port</b> of your splunk HTTP Event Collector
+                                        </div>}
+                                        bodyContent={<div>
+                                            Usually the port is 8088
+                                            See {' '}
+                                            <a
+                                                target='_blank'
+                                                rel='noreferrer'
+                                                href='https://docs.splunk.com/Documentation/Splunk/8.2.5/Data/UsetheHTTPEventCollector'>
+                                                Splunk HEC setup
+                                            </a>
+                                        </div>}
+                                    >
+                                        <button
```

Comment:
Could you use the [patternfly](https://patternfly-react.surge.sh/components/button) button?